providers/base/units/wwan/test-plan.pxu

```diff
@@ -57,7 +55,5 @@ _name: Manual wwan tests (after suspend)
 _description: Manual wwan tests for Snappy Ubuntu Core devices
 include:
     after-suspend-wwan/detect-manual
-    after-suspend-wwan/gsm-connection-manual
-    after-suspend-wwan/check-sim-present-manual
     after-suspend-wwan/scan-networks-manual
-    after-suspend-wwan/gsm-connection-interrupted-manual
+    after-suspend-wwan/gsm-connection-interrupted-manual
```

Comment:
Same as above: Both of `after-suspend-wwan/scan-networks-manual` and `after-suspend-wwan/gsm-connection-interrupted-manual` still depend on removed job `wwan/check-sim-present-manual`

Reply:
Do we still need `wwan/scan-networks-manual`?
If the WWAN connection can also prove that WWAN is able to scan the available networks, then we don't need the WWAN scan job anymore.
@pieqq What do you think?

Reply:
I'm wondering why there is no automated `scan-networks` job...

I'd rather have something separate from the connection job, so that in the test report, we can maybe check that the scanning worked, but not the connection, for instance...

Reply:
@zongminl , @rickwu666666  and @pieqq , Thank you for the suggestions. I have reverted the test cases for dependency.